sdk/formrecognizer/Azure.AI.FormRecognizer/src/AccountProperties.cs

```diff
@@ -20,11 +20,11 @@ internal AccountProperties(ModelsSummary_internal summary)
         /// <summary>
         /// The current count of trained custom models.
         /// </summary>
-        public int CustomModelCount { get; internal set; }
```

Comment:
In parallel to the main changes, removing these `internal set` properties across the client since we are not using them at all.